omod/src/main/resources/liquibase.xml

```diff
@@ -4,6 +4,12 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 
+	<changeSet id="BAH-828-001" author="Samuel34">
```

Comment:
Can you split into 2 changesets. Also should have precondition checking whether the stored procs exists or not.